ILC failing with exit code 139 for risc-v build

## Build Information
Build: https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492122&view=logs&j=c1f5742b-39fc-5fc9-82db-6047d4560106&t=0fa97341-effe-5151-3666-6fe0fbe24b92&s=ff05ad62-bb9a-53b6-ce9f-72f329a63e7c
Build error leg or test failing: Build / linux-riscv64 Debug CoreCLR_Bootstrapped
[View raw log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492122/logs/1319)
Pull request: https://github.com/dotnet/runtime/pull/130049
<!-- Error message template  -->
## Error Message

**DO NOT USE JSON BELOW IF THIS IS A BUILD BREAK** otherwise build analysis will allow pull requests to merge that break the build worse. For a build break, do not use this issue form. Make a regular new issue.

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Build%20Analysis/KnownIssueJsonStepByStep.md).

<!-- Use ErrorMessage for String.Contains matches. Use ErrorPattern for regex matches (single line/no backtracking). Set BuildRetry to `true` to retry builds with this error. Set ExcludeConsoleLog to `true` to skip helix logs analysis. -->

```json
{
  "ErrorMessage": "",
  "ErrorPattern": "error MSB3073: The command .*linux.riscv64.Checked.*ilc.*exited with code 139.",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```


<!-- Known issue validation start -->
 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492122
**Error message validated:** `[error MSB3073: The command .*linux.riscv64.Checked.*ilc.*exited with code 139.`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 7/2/2026 10:43:33 PM UTC
<!-- Known issue validation end -->
<!--Known issue error report start -->

### Report
|Build|Repository|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[1493156](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493156)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493156/logs/1780)|dotnet/runtime#100729|
|[1493548](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493548)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493548/logs/383)|dotnet/runtime#129907|
|[1493534](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493534)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493534/logs/858)|dotnet/runtime#130141|
|[1493479](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493479)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493479/logs/420)|dotnet/runtime#130159|
|[1493454](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493454)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493454/logs/414)|dotnet/runtime#130180|
|[1493435](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493435)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493435/logs/692)|dotnet/runtime#130103|
|[1493419](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493419)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493419/logs/421)|dotnet/runtime#130181|
|[1493403](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493403)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493403/logs/436)|dotnet/runtime#130156|
|[1493333](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493333)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493333/logs/885)|dotnet/runtime#124173|
|[1493324](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493324)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493324/logs/589)|dotnet/runtime#130172|
|[1493223](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493223)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493223/logs/493)|dotnet/runtime#128777|
|[1493173](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493173)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493173/logs/802)|dotnet/runtime#130173|
|[1493145](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493145)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493145/logs/697)|dotnet/runtime#130170|
|[1493075](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493075)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493075/logs/391)|dotnet/runtime#128950|
|[1493065](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493065)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493065/logs/376)|dotnet/runtime#125436|
|[1493029](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1493029)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1493029/logs/1029)|dotnet/runtime#129683|
|[1492983](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492983)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492983/logs/812)|dotnet/runtime#130167|
|[1492930](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492930)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492930/logs/480)|dotnet/runtime#130160|
|[1492919](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492919)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492919/logs/655)|dotnet/runtime#130079|
|[1492891](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492891)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492891/logs/850)||
|[1492876](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492876)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492876/logs/463)|dotnet/runtime#130161|
|[1492841](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492841)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492841/logs/513)|dotnet/runtime#129118|
|[1492732](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492732)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492732/logs/445)|dotnet/runtime#130012|
|[1492725](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492725)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492725/logs/1102)|dotnet/runtime#129683|
|[1492699](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492699)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492699/logs/431)|dotnet/runtime#129806|
|[1492674](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492674)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492674/logs/443)|dotnet/runtime#130156|
|[1492470](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492470)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492470/logs/1418)|dotnet/runtime#129442|
|[1492604](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492604)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492604/logs/982)|dotnet/runtime#128304|
|[1492598](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492598)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492598/logs/673)|dotnet/runtime#130099|
|[1492477](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492477)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492477/logs/2234)|dotnet/runtime#130140|
|[1492595](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492595)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492595/logs/663)|dotnet/runtime#130061|
|[1492578](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492578)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492578/logs/848)|dotnet/runtime#126367|
|[1492562](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492562)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492562/logs/735)|dotnet/runtime#130137|
|[1492555](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492555)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492555/logs/1181)|dotnet/runtime#130154|
|[1492544](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492544)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492544/logs/411)|dotnet/runtime#130153|
|[1492539](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492539)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492539/logs/419)|dotnet/runtime#130007|
|[1492521](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492521)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492521/logs/385)|dotnet/runtime#130151|
|[1492474](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492474)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492474/logs/431)|dotnet/runtime#130127|
|[1492454](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492454)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492454/logs/402)|dotnet/runtime#126427|
|[1492442](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492442)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492442/logs/453)|dotnet/runtime#129474|
|[1492400](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492400)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492400/logs/848)|dotnet/runtime#126367|
|[1492389](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492389)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492389/logs/839)|dotnet/runtime#130116|
|[1492365](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492365)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492365/logs/889)|dotnet/runtime#130051|
|[1492354](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492354)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492354/logs/565)|dotnet/runtime#130104|
|[1492343](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492343)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492343/logs/948)|dotnet/runtime#130142|
|[1492316](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492316)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492316/logs/984)|dotnet/runtime#130141|
|[1492315](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492315)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492315/logs/553)|dotnet/runtime#130006|
|[1492309](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492309)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492309/logs/794)|dotnet/runtime#130137|
|[1492292](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492292)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492292/logs/397)|dotnet/runtime#125436|
|[1492122](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1492122)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1492122/logs/1311)|dotnet/runtime#130049|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|50|50|50|
<!--Known issue error report end -->

--- a/src/coreclr/jit/emit.h
+++ b/src/coreclr/jit/emit.h
@@ -2818,7 +2818,7 @@ public:
     bool emitChkAlign; // perform some alignment checks
 #endif
 
-    insGroup* emitCurIG;
+    insGroup* emitCurIG = nullptr;
 
     void emitSetShortJump(instrDescJmp* id);
     void emitSetMediumJump(instrDescJmp* id);

